sdk/cosmosdb/cosmos/src/diagnostics/DiagnosticNodeInternal.ts

```diff
@@ -95,6 +95,7 @@ export class DiagnosticNodeInternal implements DiagnosticNode {
     const responseHeaders = pipelineResponse.headers.toJSON();
     const gatewayRequest = {
       activityId: responseHeaders[Constants.HttpHeaders.ActivityId],
+      correlateActivityId: requestContext.headers[Constants.HttpHeaders.CorrelatedActivityId],
```

Comment:
shouldn't we add correlatedActivityId in DiagnosticDataValue interface?

Reply:
DiagnosticDataValue interface is used to add additional information in case diagnostic mode is debug-unsafe. Its not required here.